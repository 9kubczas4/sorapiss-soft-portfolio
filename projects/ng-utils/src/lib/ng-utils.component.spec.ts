import { createComponentFactory, Spectator } from '@ngneat/spectator/jest';
import { NgUtilsComponent } from './ng-utils.component';

describe('NgUtilsComponent', () => {
  let spectator: Spectator<NgUtilsComponent>;
  const createComponent = createComponentFactory({
    component: NgUtilsComponent,
    shallow: true,
  });

  beforeEach(() => {
    spectator = createComponent();
  });

  it('should match snapshot', () => {
    expect(spectator.fixture).toMatchSnapshot();
  });
});
