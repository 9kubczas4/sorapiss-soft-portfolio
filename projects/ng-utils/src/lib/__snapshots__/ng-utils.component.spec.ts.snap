// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NgUtilsComponent should match snapshot 1`] = `
<lib-ng-utils
  __ngContext__={[Function LRootView]}
>
  <p>
    ng-utils component works!
  </p>
</lib-ng-utils>
`;
